### PR TITLE
installer runtime for el-8 - do not copy system libs into libdir

### PR DESCRIPTION
**Pull request by shrug**

### CODE REVIEW

File: configs/components/runtime-pe-installer.rb [hunk 1]
```diff
@@ -19,14 +19,14 @@
 
   else # Linux and Solaris systems
     libbase = platform.architecture =~ /64/ ? 'lib64' : 'lib'
```
Comment (mwaggett) [inline configs/components/runtime-pe-installer.rb]: looks like you can remove this too

Comment (shrug) [reply configs/components/runtime-pe-installer.rb]: yup!

Comment (shrug) [reply configs/components/runtime-pe-installer.rb]: yup!

Comment (shrug) [reply configs/components/runtime-pe-installer.rb]: yup!

Comment (shrug) [reply configs/components/runtime-pe-installer.rb]: thanks github

Comment (mwaggett) [reply configs/components/runtime-pe-installer.rb]: so enthusiastic

File: configs/components/runtime-pe-installer.rb [hunk 2]
```diff
@@ -18,15 +18,14 @@
     # Do nothing
 
   else # Linux and Solaris systems
-    libbase = platform.architecture =~ /64/ ? 'lib64' : 'lib'
-    if platform.name =~ /el-8/
-      libdir = "/usr/#{libbase}"
-    else
+\    if platform.name !~ /el-8-x86_64/
```
Comment (mwaggett) [inline configs/components/runtime-pe-installer.rb]: stray `\` ??

Comment (shrug) [reply configs/components/runtime-pe-installer.rb]: yup!

**Pull request merged**